bokehjs/src/lib/models/widgets/tables/table_column.ts

```diff
@@ -49,7 +49,7 @@ export class TableColumn extends Model {
     return {
       id: uniqueId(),
       field: this.field,
-      name: this.title,
+      name: this.title || this.field,
```

Comment:
Use new ES2020 operator `??`.

Reply:
Woah, I did not know about these js updates. I will update the code accordingly.